### ISSUE TITLE
Windows: Scrollbars to small

### ISSUE THREAD

**Issue by Hoffenbar** [giuspen/cherrytree#1299]
The vertical and horizontal scrollbars are only hardly visible, because they are little to small. Okay, they expand a little when the mouse is moving over, but moving around this way is too fiddly (for me :-)

**Comment by andezr** [comment]
I totally agree with you.
Partially mentioned this issue too here #1247

This is common issue on ubuntu
https://askubuntu.com/questions/775201/how-do-i-get-a-bigger-static-scrollbar-aka-normal-scrollbar

I **solved** this by adding System environment variable `GTK_OVERLAY_SCROLLING=0`
![Obraz 1](https://user-images.githubusercontent.com/12199354/97744848-36db2b00-1ae8-11eb-9495-325364175cad.png)

**Comment by Hoffenbar** [comment]
Yes, seems it look better with this envvar. But it should be configurable somewhetre, as it is not always a good solution to use env vars.

**Comment by ciranus** [comment]
This is a global gtk3 setting; you can play with these values for investigation:

```
scrollbar slider {
    min-width: 18px;
    min-height: 20px;
    background-color: #a3baef;
}

scrollbar slider:hover {
    min-width: 20px;
    background-color: #7ea0ef;
}
```

**Comment by janthird** [comment]
I am using Linux Mint XFCE.  My specs are as follows...
Kernel: 5.4.0-88-generic x86_64 bits: 64 compiler: gcc v: 9.3.0 Desktop: Xfce 4.16.0 
           tk: Gtk 3.24.20 wm: xfwm4 dm: LightDM Distro: Linux Mint 20.2 Uma 
           base: Ubuntu 20.04 focal 
I have edited my ~/.config/gtk-3.0/gtk.css and added...
scrollbar slider {
    /* Size of the slider */
    min-width: 20px;
    min-height: 20px;
    border-radius: 15px;
    /* Padding around the slider */
    border: 2px solid #090763;
}
Although my other GTK apps reflect this change with larger sliders, Cherrytree does not.  It still have the very thin sliders.  Anybody have any advice how I can increase the size of the sliders in Cherrytree?
